lib/src/easy_localization_app.dart

```diff
@@ -208,10 +261,13 @@ class _EasyLocalizationProvider extends InheritedWidget {
     log('Init provider', name: 'Easy Localization');
   }
 
+  /// retrieve current locale
   Locale get locale => _locale;
+  /// retrieve the locale when the locale is not in `supportedLocales`
```

Comment:
get locale if the locale is not in the list of supported locales `supportedLocales`